### PR TITLE
Remove unnecessary loggings from offline sql generation command

### PR DESCRIPTION
**Pull request by ephraimbuddy**
I found a way to remove unnecessary loggings that comes from validating revisions in the offline
migration command. Also cleaned up some codes

### CODE REVIEW

File: airflow/utils/db.py [hunk 1]
```diff
@@ -1074,9 +1073,10 @@ def _validate_version_range(command, config, version_range):
         raise AirflowException(
             'MSSQL is not supported for offline migration in Airflow versions less than 2.2.0.'
         )
-    revision = f"{REVISION_HEADS_MAP[lower]}:{REVISION_HEADS_MAP[upper]}"
+    _lower, _upper = REVISION_HEADS_MAP[lower], REVISION_HEADS_MAP[upper]
+    revision = f"{_lower}:{_upper}"
     try:
-        command.history(config, rev_range=revision)
+        list(script_.revision_map.iterate_revisions(_upper, _lower))
```
Comment (uranusjr) [inline airflow/utils/db.py]: What does this change?

Comment (ephraimbuddy) [reply airflow/utils/db.py]: Currently, if you run the `airflow db upgrade --range 2.0.0:2.2.3` it prints migration history before printing the SQL. Something like this:
```
64a7d6477aae -> e959f08ac86c, Change field in ``DagCode`` to ``MEDIUMTEXT`` for MySql
30867afad44a -> 54bebd308c5f, Adds ``trigger`` table and deferrable operator columns to task instance
e9304a3141f0 -> 30867afad44a, Rename ``concurrency`` column in ``dag`` table to`` max_active_tasks``
83f031fd9f1c -> e9304a3141f0, Make XCom primary key columns non-nullable
ccde3e26fe78 -> 83f031fd9f1c, Improve MSSQL compatibility
092435bf5d12 -> ccde3e26fe78, Add index on state, dag_id for queued ``dagrun``
```
So the above code does this internally and does not print the history.

Comment (uranusjr) [reply airflow/utils/db.py]: I see. Perhaps add a comment to describe what this line does then? (Run through the revisions to check they all execute correctly?)

File: airflow/cli/cli_parser.py [hunk 2]
```diff
@@ -244,15 +244,10 @@ def string_list_type(val):
     ('--revision-range',),
     help=(
         "Migration revision range(start:end) to use for offline sql generation. "
-        "Example: 'a13f7613ad25:7b2661a43ba3'"
+        "Example: `a13f7613ad25:7b2661a43ba3`"
```
Comment (kaxil) [inline airflow/cli/cli_parser.py]:
```suggestion
        "Example: ``a13f7613ad25:7b2661a43ba3``"
```

If this is for RST

**Review by kaxil — APPROVED** [review]

File: airflow/cli/cli_parser.py (hunk 2 above)
Comment (ephraimbuddy) [reply airflow/cli/cli_parser.py]: It was spellcheck error...

**Pull request merged**